ArduCopter/land_detector.cpp

```diff
@@ -106,6 +106,9 @@ void Copter::set_land_complete(bool b)
 
     g2.stats.set_flying(!b);
 
+    // tell AHRS land state
```

Comment:
Is this a state that is only true when in the process of landing or after we have landed, eg not flying.